app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharacterView.kt

```diff
@@ -50,4 +52,10 @@ class CharacterView(activity: MainActivity) {
         activityRef.get()!!.progressBar.visibility = View.VISIBLE
```

Comment:
use a safe call instead
activityRef.get()?.let {
    it.progressBar.visibility = View.VISIBLE
}

Reply:
done